provisioning/datasources/discovery-api.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: 1
+
+datasources:
+  - name: System73 Discovery
+    value: DS_SYSTEM73_DISCOVERY
+    type: yesoreyeram-infinity-datasource
```

Comment:
Why the need for yet another http api plugin?

You could leverage the JSON http plugin for this datasource (api) right?

Reply:
Yes but its better the other way around, as the json plugin is on maintenance mode, migrate to the infinity plugin